actioncable/lib/action_cable/channel/base.rb

```diff
@@ -202,17 +212,21 @@ def transmit(data, via: nil)
         end
 
         def defer_subscription_confirmation!
-          @defer_subscription_confirmation = true
+          @defer_subscription_confirmation_counter.increment
         end
 
         def defer_subscription_confirmation?
-          @defer_subscription_confirmation
+          @defer_subscription_confirmation_counter.value.positive?
```

Comment:
`positive?` is only available on Ruby 2.3, we need to require our core extension.